### DISCUSSION TITLE
简单主页出事了

### DISCUSSION THREAD

**Discussion by MFn233** [Meloong-Git/PCL#5011] [自定义主页]
![image](https://github.com/user-attachments/assets/ee42db89-e08d-413c-ac60-8458214c6d36)
gitee坏事做尽
所以。。。

https://[raw.gitcode.com/MFn233/PCL-Mainpage/raw/main/Custom.xaml](https://raw.gitcode.com/MFn233/PCL-Mainpage/raw/main/Custom.xaml)

**Comment by wuliaodexiaoluo** [comment]
>gitee坏事做尽

sb gitee

**Comment by new5Fpointer** [comment]
巧了，我也是。
不过网址是打得开的。
可疑的是复制代码并粘贴到自定义主页xaml后并没有任何反应
